src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -15,7 +15,6 @@ const PostgresRelationDoesNotExistError = '42P01';
 const PostgresDuplicateRelationError = '42P07';
 const PostgresDuplicateColumnError = '42701';
 const PostgresMissingColumnError = '42703';
-const PostgresDuplicateObjectError = '42710';
```

Comment:
From your issue description, it seems that this PR contains unrelated changes. Could you please move these into a separate PR?

Reply:
done